src/smart-components/request/requests.js

```diff
@@ -23,19 +22,20 @@ import useQuery from '../../utilities/use-query';
 
 const columns = [{
   title: 'Name',
-  cellFormatters: [ expandable ]
+  cellFormatters: [ expandable ],
+  transforms: [ sortable ]
 },
-'Requester',
-'Opened',
-'Updated',
-'Status',
-'Decision'
+{ title: 'Requester', transforms: [ sortable ]},
+{ title: 'Opened', transforms: [ sortable ]},
+{ title: 'Updated' },
```

Comment:
The API does not allow sorting by date?

Reply:
It allows, but there is no `updated_at` attribute

The updated is one of `finished_at`, `notified_at` (depends if `finished_at` exists)...